packages/api/taskOrderFiles/deleteTaskOrderFile.ts

```diff
@@ -0,0 +1,47 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { v4 as uuid } from "uuid";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, NoContentResponse } from "../utils/response";
+import { S3Client, DeleteObjectCommand, DeleteObjectCommandOutput } from "@aws-sdk/client-s3";
+import { FileMetadata, FileScanStatus } from "../models/FileMetadata";
+import * as parser from "lambda-multipart-parser";
+import { ErrorCodes } from "../models/Error";
+import { isPathParameterPresent } from "../utils/validation";
+
+const bucketName = process.env.PENDING_BUCKET;
+export const NO_SUCH_TASK_ORDER_FILE = new ErrorResponse(
+  { code: ErrorCodes.INVALID_INPUT, message: "TaskOrderNumber must be specified in the URL path" },
+  ErrorStatusCode.BAD_REQUEST
+);
+
+/**
+ * Delete a Task Order File
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> {
+  const taskOrderNumber = event.pathParameters?.taskOrderNumber;
```

Comment:
We updated the API a little while back to generate this ID ourselves, so it should be `taskOrderId` instead.